sos/report/plugins/tftpserver.py

```diff
@@ -21,6 +21,10 @@ class TftpServer(Plugin, IndependentPlugin):
     packages = ('tftp-server',)
 
     def setup(self):
-        self.add_dir_listing(['/tftpboot', '/sr/tftp'], recursive=True)
```

Comment:
Unrelated to your change, but while we're here - that `/sr/tftp` should surely be `/srv/tftp` (the [default location on SUSE](https://documentation.suse.com/sled/15-SP5/html/SLED-all/cha-deployment-prep-pxe.html) - would you mind adding an additional commit to your PR to put this right along with the `/var/lib/tftpboot` change?